sim_pkg/__main__.py

```diff
@@ -105,28 +59,23 @@ def main():
         vis_process.wait()
 
     except KeyboardInterrupt:
-        # Kill the processes. Currently kills the robot subprocess and everything seems to die.
-        # subprocess.poll 
+        # Kill the processes. Currently kills the robot subprocess and
+        # everything seems to die. subprocess.poll 
         for process in robot_processes:
             subprocess.Popen.terminate(process)
             stdout, stderr = process.communicate()
-            # print(stdout)
```

Comment:
Please stop leaving commented out code in your repo. We have git for that. It just makes it hard to read.
I'm new to this codebase and it takes me time to understand everything considering it's so messy. Cleaner = faster.